docs/guides/dynamo_deploy/quickstart.md

```
@@ -14,17 +14,11 @@ Use this approach when installing from pre-built helm charts and docker images p
+export RELEASE_VERSION=0.4.0
 ```
 
 Install `envsubst`, `kubectl`, `helm`
 
-### Authenticate with NGC
-
-Go to  https://ngc.nvidia.com/org to get your NGC_CLI_API_KEY.
-
-```bash
-helm repo add nvidia https://helm.ngc.nvidia.com/nvidia --username='$oauthtoken' --password=<YOUR_NGC_CLI_API_KEY>
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)